var/spack/repos/builtin/packages/py-correctionlib/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCorrectionlib(PythonPackage):
+    """A generic correction library"""
+
+    homepage = "https://github.com/cms-nanoAOD/correctionlib"
+    pypi     = "correctionlib/correctionlib-2.0.0.tar.gz"
+
+    version('2.0.0', sha256='e4d240cbdb2633a8955ddcd02d5b9bfb33d7e1a33554d6f7957f2dec56988a67')
+
+    variant('convert', default=False,
+            description='Includes select conversion routines for common types')
+
+    depends_on('python@3.6:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm', type='build')
+    depends_on('py-scikit-build', type='build')
+    depends_on('cmake', type='build')
+    depends_on('ninja', type='build')
```

Comment:
Are you sure it uses ninja? The `pyproject.toml` explicitly lists `py-make`.

Reply:
Does it?
> `requires = ["setuptools>=42", "wheel", "cmake>=3.18", "ninja"]`

The log indicates that it tries Ninja first, falling back to GNU Make if not available.